x-pack/plugins/beats/server/lib/crypto/are_tokens_equal.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { timingSafeEqual } from 'crypto';
+
+export function areTokensEqual(token1, token2) {
+  return token1.length === token2.length
```

Comment:
Are we not worried about the short-circuiting of the length evaluation disclosing when they've found the right length of token?

Reply:
Feel free to disregard this based on https://github.com/nodejs/node/issues/17178#issuecomment-348784606

Reply:
Yup, more discussion here: https://news.ycombinator.com/item?id=761059